src/index.css

```diff
@@ -20,6 +20,7 @@
   --bg-color: #1c2128;
   --text-primary: white;
   --text-secondary: black; /* TODO: Fix */
+  color:white;
```

Comment:
Also, this is not a proper way to add the colour. You should've made use of variables defined around